.chloggen/httpSD-client-config-from-TA-confighttp.yaml

```
@@ -0,0 +1,28 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: prometheusreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Use confighttp for httpSD client config
```

I feel the note does not reflect this change well. Would the following be a better description? Feel free to modify it, my understanding of this change is limited.

```suggestion
note: Ensure TLS config from Target Allocator is used in the receiver's service discovery
```

---

TLS was the main driver for this PR but as [discussed](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/33370#issuecomment-2214222263) we wanted to translate as much as possible from confighttp

---

What about
```suggestion
note: Ensure Target Allocator's confighttp is used in the receiver's service discovery
```

---

> What about

That looks clear enough to me.